### PR TITLE
sile 0.15.7

### PR DESCRIPTION
**Pull request by daeho-ro**
bump alternative
- https://github.com/Homebrew/homebrew-core/pull/199087
- https://github.com/Homebrew/homebrew-core/pull/197800

----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by alerque** [comment]
Nope—v0.15.6 and v0.15.7 were both broken on Darwin anyway as noted in other PRs. I'm going to update the original PR for v0.15.8 which should be fixed upstream now. I suggest closing this.

File: Formula/s/sile.rb [hunk 1]
```diff
@@ -15,15 +15,10 @@ class Sile < Formula
...
-  head do
-    url "https://github.com/sile-typesetter/sile.git", branch: "master"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```
[the remark this answers is not in the document]
Comment (daeho-ro) [reply Formula/s/sile.rb]: Thank you for info, I should keep the original code here :)

**Comment by daeho-ro** [comment]
Or just close by 15.8 PR?

**Comment by alerque** [comment]
> Or just close by 0.15.8 PR?

I already updated #197800 with v0.15.8. If there is a problem with that lets review it there and not fiddle with new PRs.

**Pull request closed without merging**